### PR TITLE
refactor(*): replace utility classes and css vars

### PR DESCRIPTION
**Pull request by sumimakito**
### Summary

- Replacing utility classes and CSS variables with design tokens and custom classes
- Adding ESLint rules for utility classes
- Adding Stylelint for CSS variables

KAG-2356

### CODE REVIEW

File: src/pages/plugins/PluginCardSkeleton.vue [hunk 1]
```diff
@@ -38,9 +38,31 @@ export default {
 
 <style lang="scss">
 .k-skeleton-25 {
-  --KSkeletonCardWidth: 24%;
   .skeleton-card {
     height: 240px;
   }
+
+  // --KSkeletonCardWidth: 24%;
```
Comment (Leopoldthecoder) [inline src/pages/plugins/PluginCardSkeleton.vue:45]: Can we remove this line?

**Comment by Leopoldthecoder** [comment]
Could you add `@kong/design-tokens` to the dependabot rule?

File: src/pages/plugins/PluginCardSkeleton.vue (hunk 1 above)
Comment (sumimakito) [reply src/pages/plugins/PluginCardSkeleton.vue]: Sure. This was intended to leave a trace of the equivalent variable definition in Kongponents.

**Comment by sumimakito** [comment]
@Leopoldthecoder `@kong/design-tokens` was added to dependabot rules. Thanks.

**Review by Leopoldthecoder — APPROVED** [review]
LGTM

**Pull request merged**